examples/everything/src/lib.rs

```diff
@@ -64,8 +64,12 @@ mod everything {
                 })
                 .enable_component_royalties(component_royalties! {
                     roles {
-                        royalty_admin => rule!(allow_all), updatable;
-                        royalty_admin_updater => OWNER, locked;
+                        royalty_setter => rule!(allow_all), updatable;
+                        royalty_setter_updater => OWNER, locked;
+                        royalty_locker => OWNER, updatable;
```

Comment:
It's a little weird these are set to `updatable` then the `updater` role is set to `deny_all`?